app/services/staff_action_logger.rb

```diff
@@ -46,6 +46,23 @@ def log_custom(custom_type, details = nil)
     UserHistory.create!(attrs)
   end
 
+  def edit_directory_columns_details(column_data, directory_column)
```

Comment:
Does it make sense for this helper to live in `StaffActionLogger` when it is only used by the edit directory columns controller?